Tribler/community/tunnel/tunnel_community.py

```diff
@@ -219,7 +219,7 @@ def __init__(self, community):
         self.index = -1
 
     def has_options(self, hops):
-        return len(self.community.active_data_circuits(hops)) > 0
+        return len(self.community.socks_circuits()) > 0
```

Comment:
I think he made some circuits to have some ready for downloading. Not having them ready before a download would cause problems in the socks server/libtorrent combo. That's why we would only attempt to start a download if there is a tunnel ready beforehand. For me, that makes sense